mava/systems/ff_ippo.py

```diff
@@ -357,16 +357,8 @@ def learner_setup(
         config=config, network="feedforward", centralised_critic=False
     )
 
-    actor_lr = (
-        make_learning_rate_schedule(config.system.actor_lr, config)
-        if config.system.decay_learning_rates
-        else config.system.actor_lr
-    )
-    critic_lr = (
-        make_learning_rate_schedule(config.system.critic_lr, config)
-        if config.system.decay_learning_rates
-        else config.system.critic_lr
-    )
+    actor_lr = make_learning_rate(config.system.actor_lr, config)
```

Comment:
isn't sending the config enough to retrieve the actor_lr?

Reply:
It is difficult to do, since we would then have to pass in an arg to say that we want to make a scheduler for the actor or the critic. So I felt that passing in an initial learning rate keeps things as general as possible.